entrypoint.sh

```diff
@@ -1,16 +1,16 @@
-#!/bin/bash
+#!/bin/sh
 
 set -e
 
 # Create a user to run the process as instead of root
 
-: ${SUID:=1000}
-: ${SGID:=1000}
+: "${USER_NAME:=user}"
+: "${GROUP_NAME:=usergroup}"
```

Comment:
What do `user` and `usergroup` expand to here, and how? Should they not be env vars?

Reply:
I don't think they expand to anything to be honest, I was following the previous syntax. I am happy to change these to env vars in the Dockerfile.